recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "gtk" %}
-{% set version = "4.14.1" %}
+{% set version = "4.16.5" %}
```

Comment:
The latest version is greater than this.
* 4.17 - used by gtksourceview
* 4.18.6 - https://gitlab.gnome.org/GNOME/gtk/-/releases/4.18.6
* 4.19.1 - https://gitlab.gnome.org/GNOME/gtk/-/releases/4.19.1

https://gitlab.gnome.org/GNOME/gtk/-/releases